style.css

```diff
@@ -0,0 +1,640 @@
+  border: 1px solid rgba(63, 61, 61, 0.747);
+}
+
+input[type="checkbox"]:checked {
+  background-color: rgba(89, 207, 172, 0.644);
+  border: 3px solid #2a6d55a8;
+}
+
+input[type="submit"] {
+  background-color: rgb(25, 85, 75);
+  border: none;
+  color: white;
+  padding: 10px 32px;
+  text-decoration: none;
+  margin: 2% 2%;
+  cursor: pointer;
+  transition: 0.2s ease-in;
+}
+
+input[type="submit"]:hover {
+  background-color: rgb(17, 58, 51);
+  transform: scale(95%);
+  cursor: cell;
+}
+
+
+
+
+
+
+/* ACCORDIAN */
+
+
+.accordion-div {
+  background-color: rgba(255, 255, 255, 0.39);
+  grid-area: accordion;
+  color: rgb(232, 241, 240);
+  position: relative;
+  overflow: hidden;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.accordion-form {
+  backdrop-filter: blur(10px);
+  background-color: rgba(219, 235, 231, 0.671);
+  display: block;
+  overflow: hidden;
+  margin: 1vw 10vw;
+  font-weight: bold;
+  font-family: "Lato", sans-serif;
+}
+
+.top-according {
+  background-color: #ffffff31;
+  border-bottom: 0.5px solid rgba(11, 37, 33, 0.657);
+  position: relative;
+  display: flex;
+  justify-content: center;
+  color: rgb(24, 63, 56);
+}
+
+.accordion {
+  color: rgb(24, 63, 56);
+  line-height: 170%;
+  background-color: #ffffff44;
+  position: relative;
+  border-bottom: 0.5px solid rgba(11, 37, 33, 0.418);
+  cursor: cell;
+  padding: 2%;
+}
+
+.accordion:after {
+  content: "\002B";
+  color: rgb(36, 83, 76);
+  font-weight: bold;
+  float: right;
+  padding: 0 1%;
+}
+
+.accordion.active:after {
+  content: "\2212";
+}
+
+.accordion.active {
+  border-radius: 4px 4px 0 0;
+  cursor: zoom-out;
+}
+
+.answer {
+  overflow: hidden;
+  opacity: 0;
+  max-height: 0;
+  backdrop-filter: blur(20px);
+  transition: 0.2s ease-out;
+}
+
+.answer.show {
+  opacity: 1;
+  max-height: 500px;
+  color: rgb(7, 20, 18);
+  line-height: 170%;
+  border-bottom: 0.5px solid rgba(11, 37, 33, 0.418);
+  cursor: pointer;
+  backdrop-filter: blur(20px);
+  background-color: rgba(98, 143, 135, 0.137);
+  padding: 2%;
+  transition: 0.3s ease-in-out;
+}
+
+
+
+
+
+
+
+
+/* FOOTER */
+
+
+.footer-div {
+  grid-area: footer;
+  backdrop-filter: blur(4px);
+  background-color: rgba(91, 102, 100, 0.466);
+  text-align: center;
+  overflow: hidden;
+}
+
+.footer-div p {
+  margin: 0.5%;
+  color: rgb(255, 255, 255);
+  text-shadow: 0 0 1px rgb(2, 34, 26), 0 0 2px rgb(200, 255, 242),
+    0 0 3px rgb(25, 85, 75), 0 0 5px rgb(25, 85, 75);
+  font-family: "Lato", sans-serif;
+  font-weight: bold;
+  overflow: hidden;
+  transition: 0.2s ease-in-out;
+}
+.footer-div p:hover {
+  transform: scale(93%);
+}
+
+i {
+  overflow: hidden;
+}
+.footer-div span {
+  padding: 1%;
+  text-shadow: 0 0 2px rgb(2, 34, 26);
+}
+
+.footer-div a {
+  text-decoration: none;
+  color: rgb(255, 255, 255);
+}
+
+
+
+
+
+
+
+
+
+/* MEDIA QUERIES */
+
+
+/*  PHONE  */
+
+@media (min-width: 0) and (max-width: 767px) {
```

Comment:
You may consider making the sign-up form and the accordion a little bit bigger in mobile version in order to make it more readable. :)